### ISSUE TITLE
White box is shown instead of QR code when attempting to validate login

### ISSUE THREAD

**Issue by khammo01** [make-all/tuya-local#2031]
Press Add Device. 
Provide user code when prompted. 
After pressing Submit, TuyaLocal shows the QR code screen but there is an empty white box where the QR code should go. 

Restarting Home Assistant does not resolve. 
If I try again after a few days, the code will sometimes be shown.

Please let me know if there are any logs that would be needed or steps to get the same. 

![Screenshot 2024-06-20 at 21 21 29](https://github.com/make-all/tuya-local/assets/8848588/41a24a6a-6ebb-43bf-b9f7-6d52ece9dcdc)

**Comment by make-all** [comment]
I don't think this is a bug in this integration, as a standard HA helper is used to display the QR code.
Others seem to have had the same problem with the built in tuya integration, which seems to be related to non-standard themes.

https://github.com/home-assistant/core/issues/110186#issuecomment-1938460720
